scripts/utilities/print_hists_for_proc.py

```diff
@@ -0,0 +1,12 @@
+import argparse
```

Comment:
I used to have a script to print the same information, but it could also print axes and few other things, perhaps I can add it in utilities at some point

Reply:
Sure